### ISSUE TITLE
[CI] FullClusterRestartIT.TestRollupAfterRestart failing

### ISSUE THREAD

**Issue by craigtaverner** [elastic/elasticsearch#93408]
Many tests are failing repeatedly over the last couple of days. All have the same errors:

```
java.lang.RuntimeException: An error occurred orchestrating test cluster.
Caused by: java.io.IOException: (No message provided)
at org.elasticsearch.test.cluster.util.IOUtils.deleteWithRetry0(IOUtils.java:162)
```

Other tests failing:

* [FullClusterRestartIT » testRollupAfterRestart {cluster=UPGRADED}](https://gradle-enterprise.elastic.co/s/owosjezvuspf6/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.FullClusterRestartIT/testRollupAfterRestart%20%7Bcluster=UPGRADED%7D)
* [MlConfigIndexMappingsFullClusterRestartIT » testMlConfigIndexMappingsAfterM...](https://gradle-enterprise.elastic.co/s/owosjezvuspf6/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.MlConfigIndexMappingsFullClusterRestartIT/testMlConfigIndexMappingsAfterMigration%20%7Bcluster=UPGRADED%7D)
* [FullClusterRestartIT » testNodeShutdown {cluster=UPGRADED}](https://gradle-enterprise.elastic.co/s/owosjezvuspf6/tests/:x-pack:plugin:shutdown:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.FullClusterRestartIT/testNodeShutdown%20%7Bcluster=UPGRADED%7D)
* [WatcherMappingUpdateIT » testMappingsAreUpdated {cluster=UPGRADED}](https://gradle-enterprise.elastic.co/s/owosjezvuspf6/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.WatcherMappingUpdateIT/testMappingsAreUpdated%20%7Bcluster=UPGRADED%7D)
* [CoreFullClusterRestartIT » testForbidDisableSoftDeletesOnRestore {cluster=U...](https://gradle-enterprise.elastic.co/s/owosjezvuspf6/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.CoreFullClusterRestartIT/testForbidDisableSoftDeletesOnRestore%20%7Bcluster=UPGRADED%7D)
* [MLModelDeploymentFullClusterRestartIT » testDeploymentSurvivesRestart {clus...](https://gradle-enterprise.elastic.co/s/owosjezvuspf6/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.MLModelDeploymentFullClusterRestartIT/testDeploymentSurvivesRestart%20%7Bcluster=UPGRADED%7D)
* [MlHiddenIndicesFullClusterRestartIT » testMlIndicesBecomeHidden {cluster=UP...](https://gradle-enterprise.elastic.co/s/owosjezvuspf6/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.MlHiddenIndicesFullClusterRestartIT/testMlIndicesBecomeHidden%20%7Bcluster=UPGRADED%7D)
* [QueryBuilderBWCIT » testQueryBuilderBWC {p0=UPGRADED}](https://gradle-enterprise.elastic.co/s/owosjezvuspf6/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.QueryBuilderBWCIT/testQueryBuilderBWC%20%7Bp0=UPGRADED%7D)
* [FullClusterRestartIT » testSlmPolicyAndStats {cluster=UPGRADED}](https://gradle-enterprise.elastic.co/s/fao4gm73k7blc/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.FullClusterRestartIT/testSlmPolicyAndStats%20%7Bcluster=UPGRADED%7D)
* [MlConfigIndexMappingsFullClusterRestartIT » testMlConfigIndexMappingsAfterM...](https://gradle-enterprise.elastic.co/s/fao4gm73k7blc/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.MlConfigIndexMappingsFullClusterRestartIT/testMlConfigIndexMappingsAfterMigration%20%7Bcluster=UPGRADED%7D)
* [FullClusterRestartIT » testNodeShutdown {cluster=UPGRADED}](https://gradle-enterprise.elastic.co/s/fao4gm73k7blc/tests/:x-pack:plugin:shutdown:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.FullClusterRestartIT/testNodeShutdown%20%7Bcluster=UPGRADED%7D)
* [WatcherMappingUpdateIT » testMappingsAreUpdated {cluster=UPGRADED}](https://gradle-enterprise.elastic.co/s/fao4gm73k7blc/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.WatcherMappingUpdateIT/testMappingsAreUpdated%20%7Bcluster=UPGRADED%7D)
* [CoreFullClusterRestartIT » testShrink {cluster=UPGRADED}](https://gradle-enterprise.elastic.co/s/fao4gm73k7blc/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.CoreFullClusterRestartIT/testShrink%20%7Bcluster=UPGRADED%7D)
* [MLModelDeploymentFullClusterRestartIT » testDeploymentSurvivesRestart {clus...](https://gradle-enterprise.elastic.co/s/fao4gm73k7blc/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.MLModelDeploymentFullClusterRestartIT/testDeploymentSurvivesRestart%20%7Bcluster=UPGRADED%7D)
* [MlHiddenIndicesFullClusterRestartIT » testMlIndicesBecomeHidden {cluster=UP...](https://gradle-enterprise.elastic.co/s/fao4gm73k7blc/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.MlHiddenIndicesFullClusterRestartIT/testMlIndicesBecomeHidden%20%7Bcluster=UPGRADED%7D)
* [QueryBuilderBWCIT » testQueryBuilderBWC {p0=UPGRADED}](https://gradle-enterprise.elastic.co/s/fao4gm73k7blc/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.QueryBuilderBWCIT/testQueryBuilderBWC%20%7Bp0=UPGRADED%7D)
* _and many more..._

### CI Link

https://gradle-enterprise.elastic.co/s/owosjezvuspf6/tests/:x-pack:qa:full-cluster-restart:v7.17.9%23bwcTest/org.elasticsearch.xpack.restart.FullClusterRestartIT/testRollupAfterRestart%20%7Bcluster=UPGRADED%7D

### Repro line

    ./gradlew ':x-pack:qa:full-cluster-restart:v7.17.9#bwcTest' -Dtests.class="org.elasticsearch.xpack.restart.FullClusterRestartIT" -Dtests.method="testRollupAfterRestart {cluster=UPGRADED}" -Dtests.seed=642657C8CB419BB0 -Dtests.bwc=true -Dtests.locale=es-SV -Dtests.timezone=America/Indiana/Indianapolis -Druntime.java=19

### Does it reproduce?

Yes

### Applicable branches

main

### Failure history

https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.xpack.restart.FullClusterRestartIT&tests.test=testRollupAfterRestart%20%7Bcluster%3DUPGRADED%7D

### Failure excerpt

```
java.lang.RuntimeException: An error occurred orchestrating test cluster.
Caused by: java.io.IOException: (No message provided)
at org.elasticsearch.test.cluster.util.IOUtils.deleteWithRetry0(IOUtils.java:162)
```

**Comment by craigtaverner** [comment]
Reverting the PR https://github.com/elastic/elasticsearch/pull/93062 allows the test to pass locally, so this appears to be the culprit for these failures.

**Comment by valeriy42** [comment]
I have noticed several cases where only `MlConfigIndexMappingsFullClusterRestartIT` was failing: #93504. Maybe this is related.

**Comment by mark-vieira** [comment]
This was resolved by #93486.
